templates/user_agreement.mustache

```diff
@@ -51,12 +52,12 @@
 {{/status}}
 {{^status}}
     {{#canaccept}}
-        {{#versionname}}
-            <a href="{{acceptlink}}">{{#pix}}agreedno, tool_policy, {{#str}} agreednowithlink, tool_policy, {{{versionname}}} {{/str}}{{/pix}}</a>
-        {{/versionname}}
-        {{^versionname}}
-            <a href="{{acceptlink}}">{{#pix}}agreedno, tool_policy, {{#str}} agreednowithlinkall, tool_policy {{/str}}{{/pix}}</a>
-        {{/versionname}}
+        {{#singleversion}}
+            <a href="{{acceptlink}}" data-action="acceptmodal">{{#pix}}agreedno, tool_policy, {{#str}} agreednowithlink, tool_policy, {{{versionname}}} {{/str}}{{/pix}}</a>
```

Comment:
Without looking here into the patch, I did not realize this particular status icon is clickable. Would it be better to keep it as a plain image and have an explicit link instead in the "Agreed on" column (which is empty for this case so it gives some space for it).

Reply:
(note I am not an UX expert, so take it as just an idea :-))